easybuild/easyconfigs/b/Bowtie/Bowtie-1.2.1.1-foss-2016b.eb

```diff
@@ -0,0 +1,20 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+name = 'Bowtie'
+version = '1.2.1.1'
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+ It aligns short DNA sequences (reads) to the human genome."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+sources = ['%(namelower)s-%(version)s-src.zip']
+source_urls = ['http://download.sourceforge.net/bowtie-bio/']
+
+checksums = ['71d708c957380e115ba420a96ac5f8456c6a61760a5f4dbe06305df6a42131d8']
+
+dependencies = [('tbb', '2017_U5')]
```

Comment:
@SethosII `tbb` is only a dependency for `Bowtie2`, not for `Bowtie` (at least certainly not for this version, since it hasn't been specified as such in other easyconfigs for this same `Bowtie` version)?